app/code/Magento/InventorySales/Test/Integration/Order/PlaceOrderOnNotDefaultStockTest.php

```diff
@@ -223,9 +223,9 @@ public function testPlaceOrderWithOutOffStockProductAndBackOrdersTurnedOn()
      */
     public function testPlaceOrderWithOutOffStockProductAndManageStockTurnedOff()
     {
-        $sku = 'SKU-2';
+        $sku = 'SKU-1';
         $stockId = 30;
-        $quoteItemQty = 6.5;
+        $quoteItemQty = 10.5;
```

Comment:
why did you need to change the test for a fix on UI with default checkbox value ?